app/views/spree/admin/payment_methods/_form.html.erb

```diff
@@ -58,6 +58,18 @@
 
     <% unless @object.new_record? %>
     <div class="col-12 col-md-6">
+
+    <% if @object.respond_to?(:payment_configuration_guide) %>
+      <div class="card mb-3">
+        <div class="card-header">
+          <h5 class="mb-0"><%= Spree.t(:payment_method_configuration) %></h5>
```

Comment:
@mateus-po one thing to do before merging this PR - could you add a translation key for payment_method_configuration to config/locales/en.yml ?

Reply:
Translation key added [here](https://github.com/spree/spree/pull/11991).